jwst/background/subtract_images.py

```diff
@@ -40,3 +41,44 @@ def subtract(model1, model2):
 
     # Return the subtracted model
     return output
+
+
+def subtract_ints(sci_mod, bkg_mod):
```

Comment:
Don't need this new function, as described above.

Reply:
Agreed; resolved this.